.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+### Reproduction
+
+Provide a self-contained, concise snippet of code that can be used to reproduce the issue.
+For more complex issues provide a repo with the smallest sample that reproduces the bug.
+
+Avoid including business logic or unrelated code, it makes diagnosis more difficult.
```

Comment:
Somewhere along these lines, worth mentioning that they should redact any secrets. It's easy to copy paste tokens when providing examples.

Reply:
for sure.

Updated.